src/stan/lang/generator/dump_member_var_visgen.hpp

```diff
@@ -50,13 +57,13 @@ namespace stan {
         std::vector<expression> dims = x.dims_;
         var_size_validator_(x);
         var_resizer_(x);
-        o_ << INDENT2
-           << "vals_i__ = context__.vals_i(\"" << x.name_ << "\");" << EOL;
-        o_ << INDENT2 << "pos__ = 0;" << EOL;
-        size_t indentation = 1;
+        generate_indent(indent_, o_);
+        o_ << "vals_i__ = context__.vals_i(\"" << x.name_ << "\");" << EOL;
+        generate_indent(indent_, o_);
+        o_ << "pos__ = 0;" << EOL;
+        size_t indentation = indent_;
```

Comment:
`std::size_t` is used only in the following files:

```
> find src/stan/lang -name "*.[ch]pp" -exec grep -l "std::size_t" {} \;
src/stan/lang/ast/expr_type.hpp
src/stan/lang/ast/expr_type_def.hpp
src/stan/lang/ast/fun/infer_type_indexing.hpp
src/stan/lang/ast/fun/total_dims.hpp
src/stan/lang/ast/node/statement.hpp
src/stan/lang/ast/node/var_decl.hpp
src/stan/lang/ast/node/variable.hpp
src/stan/lang/ast/variable_map.hpp
```

the generator code uses unqualified `size_t` everywhere.